app/controllers/admin/featured_policies_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Admin::FeaturedPoliciesController < Admin::BaseController
+  before_filter :load_organisation
+
+  def index
+    @featured_policies = @organisation.featured_policies.order(:ordering)
+    @policies = Policy.all.sort_by(&:title)
+  end
+
+  def create
+    @featured_policies = @organisation.featured_policies.build(policy_content_id: params[:policy_content_id])
+    if @featured_policies.save
+      redirect_to admin_organisation_featured_policies_path(@organisation), notice: 'Policies featured'
+    else
+      render :new
+    end
+  end
+
+  def destroy
+    @feature = @organisation.featured_policies.find(params[:id])
+    if @feature.destroy!
+      message = {notice: "'#{@feature.title}' unfeatured"}
+    else
+      message = {alert: "Unable to unfeature '#{@feature}' because #{@feature.errors.full_messages.to_sentence}"}
+    end
+    redirect_to admin_organisation_featured_policies_path(@organisation), message
+  end
+
+  def reorder
+    params[:ordering].each_pair do |featured_policy_id, order|
+      FeaturedPolicy.find(featured_policy_id).update_attributes(
+        ordering: order
```

Comment:
This will fit on a single line.  If you've broken it out to make possible future diffs clearer, end this with a comma.